spring-kafka/src/main/java/org/springframework/kafka/support/converter/Jackson2JavaTypeMapper.java

```diff
@@ -29,6 +29,7 @@
  * @author Sam Nelson
  * @author Andreas Asplund
  * @author Gary Russell
+ * @author Elliot Kennedy
```

Comment:
There is no any changes in this class, so, that is already speculation to add your name for nothing 😄